share/gpodder/ui/gtk/gpodderpodcastdirectory.ui

```diff
@@ -1,258 +1,417 @@
                         <property name="position">0</property>
                       </packing>
                     </child>
                     <child>
-                      <object class="GtkButton" id="btnSelectNone">
-                        <property name="label" translatable="yes">Select _none</property>
+                      <object class="GtkLabel" id="label6">
                         <property name="visible">True</property>
-                        <property name="can-focus">True</property>
-                        <property name="receives-default">False</property>
-                        <property name="use-underline">True</property>
-                        <signal name="clicked" handler="on_btnSelectNone_clicked" swapped="no"/>
+                        <property name="can-focus">False</property>
+                        <property name="margin-start">20</property>
+                        <property name="margin-top">10</property>
+                        <property name="margin-bottom">10</property>
+                        <property name="label" translatable="yes">Select a source for new podcasts</property>
+                        <property name="xalign">0</property>
+                        <property name="yalign">0</property>
                       </object>
                       <packing>
                         <property name="expand">False</property>
-                        <property name="fill">False</property>
+                        <property name="fill">True</property>
                         <property name="position">1</property>
                       </packing>
                     </child>
+                    <child>
+                      <object class="GtkTreeView" id="tv_providers">
+                        <property name="visible">True</property>
+                        <property name="can-focus">True</property>
+                        <property name="margin-start">10</property>
+                        <property name="margin-end">10</property>
+                        <property name="margin-top">10</property>
+                        <property name="margin-bottom">10</property>
+                        <property name="headers-visible">False</property>
+                        <property name="enable-search">False</property>
+                        <property name="activate-on-single-click">True</property>
+                        <child internal-child="selection">
+                          <object class="GtkTreeSelection"/>
+                        </child>
+                      </object>
+                      <packing>
+                        <property name="expand">True</property>
+                        <property name="fill">True</property>
+                        <property name="position">3</property>
+                      </packing>
+                    </child>
                   </object>
+                  <packing>
+                    <property name="name">page4</property>
```

Comment:
What are the page1, page2, page3 and page4 translatable names for?

In `lb_search` there is a translatable "label", is that shown in the UI?

Reply:
the title would appear in a gtk stack switcher if we used one. I've put some informative text because it might help screen readers.